apps/site/wrangler.jsonc

```diff
@@ -33,10 +33,24 @@
     "fs": "./.wrangler/fs-assets-polyfilling/polyfills/node/fs.ts",
     "fs/promises": "./.wrangler/fs-assets-polyfilling/polyfills/node/fs/promises.ts",
   },
-  "kv_namespaces": [
```

Comment:
I think we explain this somewhere in the docs, can we update that file if so?

Reply:
+1